addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -43,14 +43,14 @@
                                     <div class="o_Attachment_action o_Attachment_actionUnlink"
                                         t-att-class="{
                                             'o-pretty': attachment.isLinkedToComposer,
-                                        }" t-on-click="_onClickUnlink" title="Remove"
+                                        }" tabindex="0" aria-label="Remove" role="menuitem" t-on-click="_onClickUnlink" title="Remove"
```

Comment:
side note, but to take into account probably in master only: the "focus" style for those newly focusable elements when using keyboard navigation in standard dekstop is not looking good at all

and key press are not actually supported

Reply:
Agree